.github/workflows/scripts/get-plugin-version.js

```diff
@@ -0,0 +1,13 @@
+const { readFile } = require( 'fs/promises' );
+const { join } = require( 'path' );
+
+exports.getVersion = async plugin => {
```

Comment:
It feels like we should have a utility for this. Does this type of thing help? https://github.com/woocommerce/woocommerce/pull/34951

Reply:
Thanks. I initially tried to get it going with that, but I was unable. Do you think we should wait on #34951 to be merged and rework this or maybe we can revisit this and clean it up some after #34951 lands?